go/consensus/tendermint/apps/staking/transactions.go

```diff
@@ -37,20 +31,17 @@ func (app *stakingApplication) transfer(ctx *abci.Context, state *stakingState.M
 	if err != nil {
 		return err
 	}
-	if err = ctx.Gas().UseGas(1, staking.GasOpTransfer, params.GasCosts); err != nil {
+	if err := ctx.Gas().UseGas(1, staking.GasOpTransfer, params.GasCosts); err != nil {
```

Comment:
Shouldn't need this change, `err` is already declared.

Reply:
ok